doc/source/ray-overview/examples.rst

```diff
@@ -32,1596 +32,1485 @@ Ray Examples
+        How Ray solves common production challenges for generative AI infrastructure
 
     .. grid-item-card:: :bdg-primary:`Blog`
         :class-item: gallery-item training llm nlp
+        :link: https://www.anyscale.com/blog/training-175b-parameter-language-models-at-1000-gpu-scale-with-alpa-and-ray
 
-        .. button-link:: https://www.anyscale.com/blog/training-175b-parameter-language-models-at-1000-gpu-scale-with-alpa-and-ray
-
-            Training 175B Parameter Language Models at 1000 GPU scale with Alpa and Ray
+        Training 175B Parameter Language Models at 1000 GPU scale with Alpa and Ray
 
     .. grid-item-card:: :bdg-primary:`Blog`
         :class-item: gallery-item llm
+        :link: https://www.anyscale.com/blog/faster-stable-diffusion-fine-tuning-with-ray-air
 
-        .. button-link:: https://www.anyscale.com/blog/faster-stable-diffusion-fine-tuning-with-ray-air
-
-            Faster stable diffusion fine-tuning with Ray AIR
+        Faster stable diffusion fine-tuning with Ray AIR
 
     .. grid-item-card:: :bdg-primary:`Blog`
         :class-item: gallery-item training serving huggingface llm
+        :link: https://www.anyscale.com/blog/how-to-fine-tune-and-serve-llms-simply-quickly-and-cost-effectively-using
 
-        .. button-link:: https://www.anyscale.com/blog/how-to-fine-tune-and-serve-llms-simply-quickly-and-cost-effectively-using
-
-            How to fine tune and serve LLMs simply, quickly and cost effectively using Ray + DeepSpeed + HuggingFace
+        How to fine tune and serve LLMs simply, quickly and cost effectively using Ray + DeepSpeed + HuggingFace
 
     .. grid-item-card:: :bdg-primary:`Blog`
         :class-item: gallery-item llm
+        :link: https://www.businessinsider.com/openai-chatgpt-trained-on-anyscale-ray-generative-lifelike-ai-models-2022-12
 
-        .. button-link:: https://www.businessinsider.com/openai-chatgpt-trained-on-anyscale-ray-generative-lifelike-ai-models-2022-12
-
-            How OpenAI Uses Ray to Train Tools like ChatGPT
+        How OpenAI Uses Ray to Train Tools like ChatGPT
 
     .. grid-item-card:: :bdg-secondary:`Code example`
         :class-item: gallery-item llm
+        :link: /ray-air/examples/gptj_deepspeed_fine_tuning
+        :link-type: doc
 
-        .. button-ref:: /ray-air/examples/gptj_deepspeed_fine_tuning
-
-            GPT-J-6B Fine-Tuning with Ray AIR and DeepSpeed
+        GPT-J-6B Fine-Tuning with Ray AIR and DeepSpeed
 
     .. grid-item-card:: :bdg-secondary:`Code example`
         :class-item: gallery-item llm
+        :link: https://github.com/ray-project/aviary/
 
-        .. button-link:: https://github.com/ray-project/aviary/
-
-            Aviary toolkit serving live traffic for LLMs 
+        Aviary toolkit serving live traffic for LLMs
 
     .. grid-item-card:: :bdg-success:`Tutorial`
         :class-item: gallery-item pytorch
+        :link: /ray-air/examples/convert_existing_pytorch_code_to_ray_air
+        :link-type: doc
```

Comment:
Out of curiosity is there a way to combine this into a template of sorts so people don't accidentally miss this option when adding a new example?

Reply:
Maybe. Annoyingly if you set the wrong link type for the card, it will not link correctly. If you refer to another doc, the link type needs to be doc, if it's a ref the link type needs to be ref, etc. By contrast if you add a button-ref it doesn't care, it just works. So it's kind of easy to get wrong. I'm not sure if there's a way to easily set the link type as part of a template. 🤔